app/styles/_variables.less

```diff
@@ -44,17 +52,16 @@
 @log-scroll-offset-bottom: @log-scroll-offset-top;
 @log-scroll-offset-right: 25px;
 @log-scroll-offset-top: 10px;
-@log-scroll-top-offset-right-xs: (@log-scroll-offset-right + @middle-content-container-padding);
-@log-scroll-top-offset-right-sm: (@log-scroll-offset-right + @scrollbar-width + @middle-content-container-padding);
-@log-scroll-top-offset-right-lg: (@log-scroll-offset-right + @scrollbar-width + @middle-content-container-padding-lg);
+@log-scroll-top-offset-right: (@log-scroll-offset-right + @middle-content-container-padding);
 // The log viewer needs to be udpated if this value changes.  It accounts for
 // bottom margin when resizing the log in JavaScript.
 @middle-content-bottom-margin: @grid-gutter-width;
 @middle-content-container-padding: floor((@grid-gutter-width / 2)); // 20px bootstrap default
 @middle-content-container-padding-lg: (@middle-content-container-padding + 10px);
 @navbar-header-offset: 2px;
-@navbar-os-header-height-desktop: 60px;
-@navbar-os-header-height-mobile: 46px;
+@navbar-header-right-margin: 10px;
+@navbar-os-header-desktop: 59px;
+@navbar-os-header-mobile: 39px;
```

Comment:
Why remove `height` from the var name?

Reply:
can't recall why  🤔  but shouldn't have been.